app.js

```diff
@@ -41,6 +41,12 @@ app.get('/crit/:crit_id', (req, res) => {
 	});
 	
 });
+//welcome page route
+app.get('/Welcome', (req, res) => {
+	res.render('NEWUSER');})

```

Comment:
Don't use uppercase for your template file names. `'newuser'` instead of `'NEWUSER'`